src/python/WMComponent/JobSubmitter/JobSubmitAPI.py

```diff
@@ -4,21 +4,21 @@
 from WMCore.Services.PyCondor.PyCondorAPI import getScheddParamValue
 
 
-def numOfJobsSubmitUptoScheddLimit(dbi, logger=logging, condorFraction=1):
+def availableScheddSlots(dbi, logger=logging, condorFraction=1):
     """
-    check executiing jobs and campare with condor limit.
+    check executiing jobs and compare with condor limit.
```

Comment:
and since you'll have to fix the typo below, how about "executiing" and "differece"